harmony-kotlin/src/jvmAndroidCommon/kotlin/com/harmony/kotlin/ext/ByteArrayExtensions.kt

```diff
@@ -1,13 +1,15 @@
 package com.harmony.kotlin.ext
 
+import java.util.Locale
+
 /**
  * Transforms the [ByteArray] into a hexadecimal [String]
  * @return a string with the hexadecimal representation of the ByteArray
  */
 fun ByteArray.toHexString(): String {
   val sb = StringBuilder()
   for (aByte in this) {
-    sb.append(String.format("%02x", aByte))
+    sb.append(String.format(Locale.UK, "%02x", aByte))
```

Comment:
From Detekt documentation about Default locale: "if locale with tag ar-SA-u-nu-arab is a current default then %d placeholders will be evaluated to a number consisting of Eastern-Arabic (non-ASCII) digits"
Usually it is good to specify any concrete locale to avoid unexpected behaviours.

About why UK instead US or any other, it is the one that the project uses in some other places like DateExtensions.kt

Reply:
Ok, make sense